build.sbt

```diff
@@ -412,15 +415,48 @@ lazy val ratSettings = Seq(
   ratFailBinaries := true
 )
 
+/**
+ * Used on projects where we want javadoc even though the code is written in scala.
+ */
+def withScalaDocConvertedToJavaDoc(p: Project): Project = {
+  p.enablePlugins(GenJavadocPlugin)
+    .settings(unidocGenjavadocVersion := Dependencies.genjavadoc)
+}
```

Comment:
I think we could actually just move the unidocGenjavadocVersion setting to commonSettings, then any projects that enable GenJavadocPlugin will automatically get it. And since enablePlugins is the only thing this does, we can just move `enablePlugins(GenJavadocPlugin)` to the individual projects.